Source/Core/Core/PowerPC/Interpreter/Interpreter_Integer.cpp

```diff
@@ -220,16 +220,15 @@ void Interpreter::cmp(UGeckoInstruction _inst)
 {
 	s32 a = (s32)rGPR[_inst.RA];
 	s32 b = (s32)rGPR[_inst.RB];
-	int fTemp = 0x8; // a < b
+	int fTemp = 0x8; //  Assume a < b
```

Comment:
This is a really confusing way to write this, and adding the word "assume" doesn't make it any more clear.  Just move the fTemp = 0x8 into an else clause.